R/pin-meta.R

```diff
@@ -133,6 +135,9 @@ test_api_meta <- function(board) {
     testthat::expect_snapshot_error(
```

Comment:
I feel like this test could be run just once for `pin_write()` since the code doesn't vary across backends?

Reply:
Ah, great point, yep 👍